CAIPs/caip-341.md

```diff
@@ -0,0 +1,119 @@
+---
+caip: 341
+title: Extension ID Target Type Specification
+author: [Joao Tavares] (@ffmcgee725)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/341
+status: Draft
+type: Standard
+created: 2024-12-12
+requires: 294
+---
+
+## Simple Summary
+
+CAIP-341 defines the `extensionId` type as a valid target type for establishing connections with browser extension wallets.
```

Comment:
```suggestion
CAIP-341 defines the `extensionId` type as a valid target type for establishing connections with browser extension wallets via the [CAIP-294] `wallet_announce` wallet discovery event.
```

Reply:
resolved in [5b0b6f4](https://github.com/ChainAgnostic/CAIPs/pull/341/commits/5b0b6f465b810ce42d316705db7f1e52302aa7cf)